Source/FullScreenPokemon.ts

```diff
@@ -4975,7 +4975,9 @@ module FullScreenPokemon {
         partyActivateCheckThing(player: IPlayer, pokemon: IPokemon, move: IMoveSchema): void {
             var borderedThing: IThing = player.bordering[player.direction];
```

Comment:
There's no need to have `borderedThing` if `move.characterName === undefined`. Move that case before this.